.storybook/stories.test.js

```diff
@@ -0,0 +1,13 @@
+import initStoryshots from '@storybook/addon-storyshots';
+import { puppeteerTest } from '@storybook/addon-storyshots-puppeteer';
+import path from 'path';
+
+initStoryshots( {
+	suite: 'Puppeteer storyshots',
+	test: puppeteerTest( {
+		// eslint-disable-next-line sitekit/acronym-case
+		storybookUrl: `file://${ path.resolve( __dirname, '../dist' ) }`,
+		setupTimeout: 5000,
+		testTimeout: 5000,
```

Comment:
To check! are these millseconds or seconds :thinking: 

The docs aren't clear

Reply:
It looks like milliseconds from inspecting the code (docs don't say)

https://github.com/storybookjs/storybook/search?q=setupTimeout

![looks like ms](https://user-images.githubusercontent.com/8496063/128050599-46b0be93-663d-47a5-8171-c6079a7bdf87.png)

Reply:
I would expect milliseconds as well.